lib/ui/overflow_menu.dart

```diff
@@ -33,19 +33,31 @@ class _OverflowMenuState extends ConsumerState<OverflowMenu> {
 
   @override
   Widget build(BuildContext context) {
-    return PopupMenuButton<OverflowMenuOption>(
-      color: Theme.of(context).colorScheme.background,
-      icon: const Icon(Icons.more_vert),
-      shape: RoundedRectangleBorder(
-        side: const BorderSide(),
-        borderRadius: BorderRadius.circular(20),
+    return Theme(
+      data: Theme.of(context)
+          .copyWith(useMaterial3: false), // disable for shadow to work
+      child: PopupMenuButton<OverflowMenuOption>(
+        color: Theme.of(context).colorScheme.background,
+        icon: const Icon(Icons.more_vert),
+        elevation: 7.0,
+        shape: RoundedRectangleBorder(
+          side: BorderSide(
+              width: 0, color: Theme.of(context).colorScheme.background),
+          borderRadius: BorderRadius.circular(5),
+        ),
+        onSelected: _handleSelectedOption,
+        itemBuilder: (BuildContext context) {
+          return OverflowMenuOption.values.map((option) {
+            return PopupMenuItem(
+                value: option,
+                child: Text(
+                  option.label,
+                  style: const TextStyle(
+                      fontWeight: FontWeight.w400, fontSize: 14),
```

Comment:
could we add a file for the styles we can reuse?
Similar to the color_schemes one file that holds our textStyles etc

Reply:
done

Reply:
Sorry for not being clear enough with my suggestion and it definately worked this way!
Could you please have a look at the second part of this anser?
https://stackoverflow.com/questions/57531969/declaring-a-styles-file-in-flutter
I think we should make a class that holds all our textstyles :)
Thank you!

Reply:
done